packages/kbn-management/settings/components/form/bottom_bar/index.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+
+import { EuiBottomBar, EuiButton, EuiButtonEmpty, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+interface BottomBarProps {
+  saveAll: () => void;
+  clearAllUnsaved: () => void;
+}
+
+export const BottomBar = ({ saveAll, clearAllUnsaved }: BottomBarProps) => {
+  return (
+    <EuiBottomBar>
+      <EuiFlexGroup
+        justifyContent="spaceBetween"
+        alignItems="center"
+        responsive={false}
+        gutterSize="s"
+      >
+        <EuiFlexItem />
+        <EuiFlexItem grow={false}>
+          <EuiButtonEmpty
+            color="ghost"
+            size="s"
+            iconType="cross"
+            onClick={clearAllUnsaved}
+            aria-describedby="aria-describedby.countOfUnsavedSettings"
+            data-test-subj="advancedSetting-cancelButton"
+          >
+            {i18n.translate('advancedSettings.form.cancelButtonLabel', {
+              defaultMessage: 'Cancel changes',
+            })}
+          </EuiButtonEmpty>
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiButton
+            className="mgtAdvancedSettingsForm__button"
```

Comment:
nit: let's move these classnames to Emotion.

Reply:
Added this change with https://github.com/elastic/kibana/pull/166460/commits/ad18ae37177353621d2facc1ab96c766bb9dc8f6